src/pages/registration/references_tab/reference-helpers.ts

```diff
@@ -0,0 +1,53 @@
+import { levelMap, Publisher } from '../../../types/registration.types';
+import { JournalPublicationContext } from '../../../types/publication_types/journalRegistration.types';
+import { PublicationType } from '../../../types/publicationFieldNames';
+import { BookPublicationContext } from '../../../types/publication_types/bookRegistration.types';
+
```

Comment:
veldig usikker på hvordan vi skal gjøre det som skjer i denne fila på en god måte...
men jeg mener at Autocomplete til tidsskrift, serie, etc, bør ha options av typen Publisher (som rett fra kanalregister-proxy), og at den dermed bør ha value av typen Publisher også.

problemet her er at verdiene til Publisher blir "spredd" rundt i publicationContext, så vi må mappe verdier fra Publisher til publicationContext og omvendt. i realiteten er det vel bare to versjoner vi vil mappe fra Publisher til context; enten med `publisher` eller med `title`, så tror ikke vi skal trenge mer enn dette (typene jeg har brukt nå må uansett endres når jeg legger til støtte for flere)

kanskje er det bedre å bare droppe dette fullstendig, og sette hver enkelt verdi manuelt med `setFieldValue`.

vet ikke om dette ga mening, men det vet jeg ikke om koden gjør heller så det passer egentlig godt...

Reply:
Tror jeg skjønner hva du mener. Det er veldig irriterende at du får samme objekt men bare med/uten title/publisher og med/uten seriesTitle. Jeg tror det blir bedre når vi får flatet ut datamodellen hvertfall, så jeg foreslår å bare la det være sånn her foreløpig